src/styles/blocks/card.scss

```diff
@@ -0,0 +1,21 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  font-family: Roboto, 'Helvetica Neue', Helvetica, sans-serif;
+  font-weight: 400;
+  line-height: 18px;
+  font-size: 12px;
+  padding: 32px 16px 16px;
```

Comment:
The padding values can also be moved to variables, especially if these values are shared among multiple blocks, to ensure consistency and maintainability.